### DISCUSSION TITLE
Overlapping territories and rendering order

### DISCUSSION THREAD

**Discussion by agutierr78** [maplibre/maplibre-gl-js#3778] [Q&A] [answered]
We have a case in which we have overlapping tiles in the same layer on a map, long story but basically it represents territories that in some cases include the same Zip codes.
We are ok with one territory hiding part of other when both are in the bounding box (i.e. national view), but we are trying to understand if the rendering will be consistent, meaning that if two people look at the map section where the territories overlap, they will always see it the same way.
Can anyone point to documentation or shed some light about this?

**Comment by HarelM** [comment, marked as answer]
Not sure I understand the questions fully, but the rendering logic is deterministic and different users will see the same map assuming it uses the same style.
Things that might cause differences are different zoom, location, pitch etc for labels and symbols as those are removed dynamically to reduce clutter.
Hope that answers your question.

**Reply by agutierr78** [reply]
That's what I was hoping to hear, as long as the overlap always shows in the same way, i.e. territory A hides a part of territory B when both are visible, then we are ok.

Thanks
